en/integrations/plugin-packs/procedures/applications-monitoring-centreon-mbi.md

```diff
@@ -3,122 +3,329 @@ id: applications-monitoring-centreon-mbi
+| :---------- | :----------------------------- | :--------- | :------ |
+| Poller      | Centreon MBI reporting server  | TCP        | 5666    |
+
+The monitored Centreon MBI reporting server will need two components to be 
+monitored:
+
+* the `centreon_linux_local.pl` plugin
+* the NRPE3 daemon
 
-Install the plugin on poller that will monitor Centreon MBI reporting server
+To install them, run the commands below:
 
-``` shell
-yum install centreon-nrpe-plugin centreon-plugin-Operatingsystems-Linux-Snmp
+```shell
+yum install http://yum.centreon.com/standard/20.04/el7/stable/noarch/RPMS/centreon-release-20.04-1.el7.centos.noarch.rpm
+yum install centreon-nrpe3-daemon.x86_64 centreon-plugin-Operatingsystems-Linux-Local.noarch
 ```
 
-### NRPE
+The value of the `allowed_hosts` parameter in `/etc/nrpe/centreon-nrpe3.cfg` 
+must be changed to include the pollers IP addresses:
+
+```ini
+[...]
+# ALLOWED HOST ADDRESSES
+# This is an optional comma-delimited list of IP address or hostnames
+# that are allowed to talk to the NRPE daemon. Network addresses with a bit mask
+# (i.e. 192.168.1.0/24) are also supported. Hostname wildcards are not currently
+# supported.
+allowed_hosts=127.0.0.1,::1
+[...]
+```
 
-Install NPRE on the Centreon MBI reporting server
+Futhermore, add the following lines at the end of the file:
 
-``` shell
-yum install centreon-nrpe-daemon
+```ini
+command[check_partitions]=/usr/share/centreon-bi/etl/centreonbiMonitoring.pl --partitions
+command[check_db]=/usr/share/centreon-bi/etl/centreonbiMonitoring.pl --db-content
+command[check_jobs]=/usr/share/centreon-bi/etl/centreonbiMonitoring.pl --jobs
 ```
 
-Edit the file */etc/nrpe/centreon-nrpe.cfg* and:
+And the service must be restarted:
 
-  - Add the IP of the poller that monitors the Centreon MBI reporting server on
-    the *allowed\_host=* part.
+```shell
+systemctl restart centreon-nrpe3.service
+```
 
-  - Add the following lines at the end of the file
-    
-        command[check_partitions]=/usr/share/centreon-bi/etl/centreonbiMonitoring.pl --partitions
-        command[check_db]=/usr/share/centreon-bi/etl/centreonbiMonitoring.pl --db-content
-        command[check_jobs]=/usr/share/centreon-bi/etl/centreonbiMonitoring.pl --jobs
+Install the plugin on each needed poller:
 
-Start the deamon:
+```shell
+yum install centreon-nrpe3-plugin
+```
 
-    # service centreon-nrpe start
+If everything is fine, this command:
 
-Be sure to have with you the following information:
+```shell
+/usr/lib64/nagios/plugins/check_centreon_nrpe3 -H monitored_host_ip -p 5666
+```
 
-  - Read-Only SNMP community
-  - IP Address of the monitoring server
+should produce this output:
 
-### Install the SNMP service
+```
+NRPE v3.2.1
+```
 
-With the `root` user, install the following package and its dependencies:
+Otherwise please refer to the [troubleshooting](#troubleshooting) section.
 
-    root@yourserver#&gt; yum install net-snmp net-snmp-utils
+### SNMP
```

Comment:
Same remark for SNMP, maybe just add link to send the user on the Linux SNMP monitoring Proc.